lib/App/DuckPAN/Restart.pm

```diff
@@ -58,31 +59,65 @@ sub run_restarter {
     }
 }
 
+sub _get_directories_to_monitor {
+    my $self = shift;
+    my @output_dirs;
+
+    try {
+        my $template_defs = App::DuckPAN::TemplateDefinitions->new;
+        my @templates = $template_defs->get_templates;
+
+        for my $template (@templates) {
+            push @output_dirs, $template->output_directory
+                if $template->needs_restart;
+        }
+    } catch {
+        if (/template definitions/i) {
+            # There was a problem loading the template definitions file. This
+            # can happen if the instant answer repository is of an older
+            # version and does not have the templates.yml file.
+            #
+            # In this case we use the older method of determining directories
+            # to monitor. This support can be removed sometime in the future.
+            # Note: Cheatsheet directories will not be monitored with this
+            # method.
+
+            $self->app->emit_debug("Instant Answers repository does not have a " .
```

Comment:
@moollaza Thanks, I have set the message to be shown in verbose mode because I think users don't need to be concerned about it normally. You should be able to see the message by running `duckpan -v server`.